superdesk/utils.py

```diff
@@ -154,3 +155,11 @@ def compare_preferences(original, updates):
     removed = original_keys - updates_keys
     modified = {o: (original[o], updates[o]) for o in intersect_keys if original[o] != updates[o]}
     return added, removed, modified
+
+
+def sha(text):
```

Comment:
Maybe rename this method to `sha1` to make it clear in the calling code which SHA hash this function calculates.
We should also **strongly** reconsider if we indeed want to use SHA-1 in the first place - it's considered broken (or at least will be in the near future) based on the most recent findings. We should better move to a different hashing function sooner rather than later.